integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/wait/service/ResourceAwait.java

```diff
@@ -37,7 +37,12 @@
                 client.waiterService().waitObject(new WaitFailedChecker<>(), waitObject, testContext, pollingInterval, maxRetry, maxRetryCount);
                 entity.refresh();
             } else {
-                client.waiterService().waitObject(new WaitOperationChecker<>(), waitObject, testContext, pollingInterval, maxRetry, maxRetryCount);
+                if (runningParameter != null && runningParameter.getTimeoutChecker() != null) {
```

Comment:
Could you please add this to the InstanceAwait?

Reply:
sure